### PR TITLE
[Kotlin Migration] NoSuchDeckException 

### PR DESCRIPTION
**Pull request by codingtosh**
[Kotlin Migration] NoSuchDeckException 
package: com.ichi2.libanki.exception

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/libanki/exception/NoSuchDeckException.kt [hunk 1]
```diff
@@ -13,23 +13,14 @@
...
  *  this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-
-package com.ichi2.libanki.exception;
+package com.ichi2.libanki.exception
 
 /**
  * A deck was accessed which did not exist.
- * <br/>
+ * <br></br>
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/libanki/exception/NoSuchDeckException.kt:20]: replace with newline

**Review by david-allison-1 — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/java/com/ichi2/libanki/exception/NoSuchDeckException.kt (hunk 1 above)
Comment (codingtosh) [reply AnkiDroid/src/main/java/com/ichi2/libanki/exception/NoSuchDeckException.kt]: Done, PTAL.

**Review by david-allison-1 — APPROVED** [review]
LGTM

**Comment by david-allison-1** [comment]
Note: I've added a "Good First Issue" to handle the `<br>` issue: https://github.com/ankidroid/Anki-Android/issues/10209

**Review by mikehardy — APPROVED** [review]

**Pull request merged**